### PR TITLE
Ref #9 Fixes Lexicographical issue for imports in StudyPlanner

### PR DESCRIPTION
**Pull request by cljones**
These changes puts the imports of the StudyPlanner class in order.

### CODE REVIEW

File: src/Controller/MainController.java [hunk 1]
```diff
@@ -234,6 +234,8 @@ public static boolean isNumeric(String str) {
 			Double.parseDouble(str);
 		} catch (NumberFormatException nfe) {
 			return false;
+		} catch (NullPointerException nfe) {
```
Comment (rsanchez-wsu) [inline src/Controller/MainController.java]: This does not seem related related to the import ordering references in the pull request summary.  Additionally, we previously discussed why this change is not appropriate in this method.

Comment (cljones) [reply src/Controller/MainController.java]: I meant to only commit the changes concerning the imports for the Study Planner.
